src/pipeline/rackhd/docker/Build.groovy

```diff
@@ -0,0 +1,41 @@
+package pipeline.rackhd.docker
+
+def build(String manifest_path, String clone_dir, String workdir){
+    def ret_dict=[:]
+    node(build_docker_node){
+            deleteDir()
+            def current_workspace = pwd()
+            String library_dir = "./on-build-config"
+            def share_method = new pipeline.common.ShareMethod()
+            share_method.checkoutOnBuildConfig(library_dir)
+            withCredentials([
+                usernamePassword(credentialsId: 'ff7ab8d2-e678-41ef-a46b-dd0e780030e1',
+                                 passwordVariable: 'SUDO_PASSWORD',
+                                 usernameVariable: 'SUDO_USER')]){
+                    timeout(90){
+                        withEnv(["WORKSPACE=${current_workspace}"]){
+                            sh """#!/bin/bash -ex
+                            bash $library_dir/src/pipeline/rackhd/docker/build.sh \
+                                   --MANIFEST_FILE_URL  ${manifest_path} \
+                                   --ARTIFACTORY_URL ${ARTIFACTORY_URL} \
+                                   --STAGE_REPO_NAME ${STAGE_REPO_NAME} \
+                                   --CLONE_DIR ${clone_dir} \
```

Comment:
How about define the parameter in this function rather than arguments of the function?
Because we are building docker images on a new node.
The father step of build() doesn't care the ```clone_dir ``` and ```workdir```